### PR TITLE
#40: Localization added for help button

### PR DESCRIPTION
**Pull request by dsuren1**
## Description
This PR adds localization data added for help button

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [x] Bugfix

## Issue

**What is the current behavior?**
#40 

**What is the new behavior?**
Help button displays correct translation text as tooltip

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [ ] Yes, and I documented them in migration notes
 - [x] No

### CODE REVIEW

File: assets/translations/data.fr-FR.json [hunk 1]
```diff
@@ -28,7 +28,8 @@
             },
             "landPlanning" : {
                 "print": "Impression",
-                "cancel": "Annuler"
+                "cancel": "Annuler",
+                "help": "Aider"
```
Comment (MaelREBOUX) [inline assets/translations/data.fr-FR.json]: "Aide" would be better

File: js/extension/constants.js [hunk 2]
```diff
@@ -13,8 +13,7 @@ export const DEFAULT_CADASTRAPP_URL = "/cadastrapp/services";
 export const DEFAULT_URBANISMEAPP_URL = "/urbanisme";
 export const URBANISME_TOOLS = {
     NRU: "NRU",
-    ADS: "ADS",
-    HELP: "HELP"
+    ADS: "ADS"
```
Comment (MaelREBOUX) [inline js/extension/constants.js:16]: Sure ?

**Review by MaelREBOUX — COMMENTED** [review]
I made 2 comments

File: js/extension/constants.js (hunk 2 above)
Comment (dsuren1) [reply js/extension/constants.js:16]: Removed unused constant

**Review by offtherailz — APPROVED** [review]

**Review by MaelREBOUX — APPROVED** [review]

**Pull request merged**